src/AzureStack/Admin/ComputeAdmin/Compute.Admin/Properties/AssemblyInfo.cs

```diff
@@ -14,5 +14,5 @@
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
 
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.2.0.0")]
+[assembly: AssemblyFileVersion("0.2.0.0")]
```

Comment:
Don't change this!

Reply:
@dsgouda Should this file be changed?  I have heard conflicting information!

Reply:
For a version bump yes.
The guideline is:
If a major version is bumped, i.e., from `0.0.0` to `1.0.0`, both `AssemblyVersion` and `AssemblyFileVersion` must be changed.
If a minor/patch version is bumped, only the `AssemblyFileVersion` must be bumped, i.e., `0.1.0` to `0.2.0`
tagging @shahabhijeet

Reply:
Thanks for the clarification!